### PR TITLE
Adding review form

### PR DESCRIPTION
**Pull request by BRS7**
added review form

### CODE REVIEW

File: app/controllers/reviews_controller.rb [hunk 1]
```diff
@@ -1,3 +1,32 @@
...
+  def new
+    @restaurant = Restaurant.find(params[:restaurant_id])
+    @review = Review.new
+  end
+
+  def create
+    @restaurant = Restaurant.find(params[:restaurant_id])
+
+    @review = Review.new(review_params)
+    @review.user_id = current_user.id
```
Comment (dpickett) [inline app/controllers/reviews_controller.rb]: you can also just `@review.user = current_user`

File: spec/features/user_adds_new_review_spec.rb [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    fill_in "Description", with: "man do i hate this place"
+    fill_in "Rating", with: "1"
+    fill_in "Price", with: "2"
+
+    click_button "Add Review"
+
+    expect(page).to have_content "Review added successfully"
+    expect(page).to have_content r1.name
+    expect(page).to have_content 1
+    expect(page).to have_content "man do i hate this place"
```
Comment (dpickett) [inline spec/features/user_adds_new_review_spec.rb]: lol

File: app/controllers/reviews_controller.rb (hunk 1 above)
Comment (dpickett) [reply app/controllers/reviews_controller.rb]: what happens when I'm not signed in? 

We should check in our controller to ensure that a user is signed in prior to presenting them with the review form or allowing them to invoke the `create` action. 

Check out our article on authorization: https://learn.launchacademy.com/lessons/devise-authorization

Comment (nwalberts) [reply app/controllers/reviews_controller.rb]: Still not seeing this change

**Pull request merged**